### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by amrita019**

### CODE REVIEW

File: node-backend/server.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var express = require("express");
```
Comment (rajikaimal) [inline node-backend/server.js]: Let's use `const` and `let` for all variable declarations.

File: node-backend/helpers/utils.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var jwt = require("jsonwebtoken");
+require('dotenv').config();
+
+
+module.exports = {
```
Comment (rajikaimal) [inline node-backend/helpers/utils.js]: Format code with `Prettier`

File: node-backend/server.js (hunk 1 above)
Comment (amrita019) [reply node-backend/server.js]: Okay

**Comment by rajikaimal** [comment]
@amrita019 Please rename react-frontend to client.

File: client/src/_services/user.services.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+function register(user) {
+  const requestOptions = {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(user)
+  };
+
+  return fetch(`http://localhost:4000/signup`, requestOptions).then(
```
Comment (rajikaimal) [inline client/src/_services/user.services.js]: Let's move URLs to different file.

File: server/routes/auth.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+let express = require("express");
```
Comment (rajikaimal) [inline server/routes/auth.js]: Change these variables to const

File: client/src/_services/user.services.js (hunk 3 above)
Comment (amrita019) [reply client/src/_services/user.services.js]: Sure

**Pull request merged**